projects/1.20/assets/ohoy-scallywag/ohsfennx/lang/zh_cn.json

```
@@ -0,0 +1,24 @@
+{
+    "item.ohsfennx.rum": "朗姆酒",
+    "item.ohsfennx.raw_ham": "生火腿",
+    "item.ohsfennx.mystery_treasure_charm": "神秘宝藏符咒",
+    "item.ohsfennx.speed_treasure_charm": "速度宝藏符咒",
+    "item.ohsfennx.luck_treasure_charm": "幸运宝藏符咒",
+    "item.ohsfennx.damage_treasure_charm": "伤害宝藏符咒",
+    "item.ohsfennx.eye_patch": "眼罩",
+    "item.ohsfennx.flintlock": "火石枪",
+    "item.ohsfennx.cutlass": "短刀",
+    "item.ohsfennx.pirate_spawn_egg": "海盗生成蛋",
+    "item.ohsfennx.pirate_captain_spawn_egg": "海盗船长生成蛋",
+    "item.ohsfennx.pirate_skeleton_king_spawn_egg": "海盗骷髅王生成蛋",
+    "item.ohsfennx.pirate_skeleton_spawn_egg": "海盗骷髅生成蛋",

```

```suggestion
    "item.ohsfennx.pirate_spawn_egg": "海盗刷怪蛋",
    "item.ohsfennx.pirate_captain_spawn_egg": "海盗船长刷怪蛋",
    "item.ohsfennx.pirate_skeleton_king_spawn_egg": "海盗骷髅王刷怪蛋",
    "item.ohsfennx.pirate_skeleton_spawn_egg": "海盗骷髅刷怪蛋",
```
请注意参照原版术语`刷怪蛋`/`Spawn Egg`